wikiconv/conversation_reconstruction/construct_utils/utils/third_party/__init__.py

```diff
@@ -1,7 +1 @@
 # -*- coding: utf-8 -*-
-
-from __future__ import (absolute_import, division, print_function,
-                        unicode_literals)
-from builtins import (bytes, dict, int, list, object, range, str, ascii, chr,
-                      hex, input, next, oct, open, pow, round, super, filter,
-                      map, zip)
```

Comment:
What's with this change?

Reply:
It's a meaningless file with unnecessary includes, no longer needed with bazel. Reverted for now will address in a future change.